modules/ROOT/pages/troubleshoot-api-provisioning-issues.adoc

```diff
@@ -52,3 +56,45 @@ To resolve this issue:
 . Delete the API.
 . Resolve the username and password error.
 . Re-create the API.  
+
+
+[[provision-call-failed]]
+== Provision call failed
+
+When you create an API, the Asset(in exchange) and/or API(in API Manager) are not successfuly created on the Anypoint Platform
+
+== Causes
+
+This error can occur if:
+ 
+* You are trying to re-create an asset on exchange after having deleted it more than seven days ago.
```

Comment:
This is not the problem, if you create an asset, delete it immediately and try to use it again after 7 days it should work as expected.

The real problem is if you had that asset **live** for more than 7 days and then you deleted it. 

```suggestion
* You are trying to re-create an asset on Exchange that existed more than seven days before being deleted.
```